### DISCUSSION TITLE
Transaction stuck after being approved on ETH blockchain

### DISCUSSION THREAD

**Discussion by jetlej** [wormhole-foundation/wormhole#443] [Problems / bugs] [answered]
I used Wormhole to transfer 7WETH from Ethereum to Solana. It got all the way through to the ETH 15 confirmations, and then never was confirmed via Solana. 

Can you please help? This is a very sizable amount of money...

**Comment by leoluk** [comment, marked as answer]
Looks like the token account wasn't created, likely due to the Solana outage: https://explorer.solana.com/address/FrCXM2SCjMNPhw8RAhq3jDy8EE34q5NbLmBHS5EwURjD

Please follow yamijuan's advice here to create the missing account: https://github.com/solana-labs/oyster/issues/196#issuecomment-909535731. Let me know once the account exists.

---

Reference for myself: 

> 01000000010d00b4907a26bd0fb52e4594b7436da4ec7d1209fbd52602b07b2b4f74b34d0994125849eac0d513e587e23a2bb814bf069136b7ef52c72992eea63c8b19860fc2ce0101c925758f9db0e6d9ec5b55b1357a385a185ccb9a61e7c8bdb98fd0837921cbac7adfe0723b123c96eaf37a5a94cfd10ba19f0be408571876ce50072732ffbc600102068534ebcecc3dbcf27227b1f3e21741ee8058a208fe884d0718f451c8f39bb83213f3d25733d4c2c17fa45b5a4cc15541dc5eed63c12167b95e19c79ea60f4e0104ed3780afd2ace7082aad0910952ac90056930dab6d3bc61acda4b2d6a709886f38a2620db106003bf088ae74927616a0fc7d433b05d1411ec34b0d0ff8aa50b70005d84623d1cb997025df077ae8bb5178387041a1147449ee2e0da7a6005f6b944e4d349ad1f2c0e6cdf9cc0e7329423b282c4e02d7169a7b9a7c4e2e9b9265dbf7000699dd915291f37d6674b854079f22da85143a1981e9f76615c6969e54a87104fd1868839a4dc51876adcb9c45df11de9aa9cc452d28cfe93b63f395d01092526101072dd39c3a141059896fd4da14532f17f947727474be8b4e66214cb83b8eb98a527e7920c4a0c1be19373c25d672072c8b70df7c7cdc8af1c661aa95e3002912f50008a6a382e17fa25f284a418e9d89cca7cffe171eaf6102275ff4ef40f84ef164c0079ed973d486c3b502bcc5aa6ff2bacb9005d8d82d2c56a1b15b9152c49ec8c2000a90f0c2977574cb5bedcbe63e165d0f0024b7149ff4d9a6c7b1a4000e6518823b325aaab7d56a31b434cbf11f204a401e423238bc9d6d2e450956c591479104c0000d5173b9b37fc12354bdc50341cdc1da9764099e13c87a2d3636cb7e8f19f2978c3409d4caee70d2b339c5b6e9a26a2579e2bc580da396108ed8fea41cba25341e010f20f82bb484062e0d73920196179fa2f0c9a11b030fab934afae8939807ff5bae51623a740c817e181e844925f2af5cd9a36445f9d3e95b8952dc18264a21e1180110181484a563a3bb9b1cffcc7a52f3af33eaf8ea85117cd0aeb14547e26a147e45580e2ceff8b39d5c7434157775378f0a16271691bd4bd6ba9d3c76b59e4337570112527b173e86b61de3b78f0d602e8c0627a7f3443609cdee31eea82cf657caa0645cafb2f94d043de87e098c97cb700c8c22dd7c4cca6d0682aeb811398f8b9881016141246010000000860201000000000000000000000000f9de20a6053d7e0ae2d906cce328935bcc45bbacdc9e1b1e4549d40f961e967fe87e229ab3d654e826fe9548e1486de2f9b25d2802000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20900000000000000000000000000000000000000000000000000000001a72b30c4

**Reply by jetlej** [reply]
Hey thanks for the quick reply! I initiated the transfer of a small amount of WETH. There was no popup from Solana / Phantom though. It went straight to MetaMask. I just sent the transaction through for good measure.

**Reply by jetlej** [reply]
Any update on this?

**Reply by leoluk** [reply]
It worked!

https://explorer.solana.com/tx/3S7yUCj9vZvNe5fMT7aC3DPYmZ2aasqKMZZ7wzFVQfhQ6khD7bcpWwHkZNUXEQzGGHKpN9eGyawDs1MDjEcDor5s

**Reply by jetlej** [reply]
Thank you so much!
